src/main/java/at/hannibal2/skyhanni/config/features/rift/PunchcardConfig.java

```diff
@@ -0,0 +1,48 @@
+package at.hannibal2.skyhanni.config.features.rift;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.config.core.config.Position;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorColour;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigLink;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+import io.github.notenoughupdates.moulconfig.observer.Property;
+
+public class PunchcardConfig {
+    @Expose
+    @ConfigOption(name = "Enabled", desc = "Highlights unpunched players in the Rift.")
```

Comment:
Since this is no longer the only feature toggle, please rename to "highlight" or something